src/Coordination/KeeperServer.cpp

```diff
@@ -273,6 +273,13 @@ void KeeperServer::launchRaftServer(const Poco::Util::AbstractConfiguration & co
         coordination_settings->election_timeout_lower_bound_ms.totalMilliseconds(), "election_timeout_lower_bound_ms", log);
     params.election_timeout_upper_bound_ = getValueOrMaxInt32AndLogWarning(
         coordination_settings->election_timeout_upper_bound_ms.totalMilliseconds(), "election_timeout_upper_bound_ms", log);
+
+    if (params.election_timeout_lower_bound_ >= params.election_timeout_upper_bound_)
```

Comment:
Is it make sense to continue work? May be terminate execution?

For example, python has some restriction on arguments, if they break, programs fails to start.

Reply:
It is interesting. That code is triggered in our Fast Tests.

https://s3.amazonaws.com/clickhouse-test-reports/46274/fff2c09b87ce319c3b9e62c22cdf19a444a4c2fe/fast_test/server.log
`2023.02.13 12:42:27.884239 [ 19622 ] {} <Fatal> KeeperServer: election_timeout_lower_bound_ms is greater than election_timeout_upper_bound_ms, this would disable leader election completely.`

tests/config/config.d/keeper_port.xml 
```
            <!-- For instant start in single node configuration -->
            <heart_beat_interval_ms>0</heart_beat_interval_ms>
            <election_timeout_lower_bound_ms>0</election_timeout_lower_bound_ms>
            <election_timeout_upper_bound_ms>0</election_timeout_upper_bound_ms>
```

As I understand, that zero values are set just because there is the only one instance and the values are needless.

Reply:
I propose just pass zero values. The defaults are sensible 1s and 2s.